docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+          Type:                    Ready
+        Last Server Node Started:  2022-01-31T17:35:39Z
+        Node Statuses:
+          demo-dc1-default-sts-0:
+            Host ID:  61dfa8cc-2a8b-4e8f-ae82-01c51833e0ba
+          demo-dc1-default-sts-1:
+            Host ID:  369aa179-d96e-4f21-a893-f6e6dc84b396
+          demo-dc1-default-sts-2:
+            Host ID:          bbdb6a9a-063b-4565-9704-f4caa6fd80f1
+        Observed Generation:  1
+        Quiet Period:         2022-01-31T17:37:06Z
+        Super User Upserted:  2022-01-31T17:37:00Z
+        Users Upserted:       2022-01-31T17:37:00Z
+      Stargate:
+        Available Replicas:  1
+        Conditions:
+          Last Transition Time:  2022-01-31T17:37:48Z
+          Status:                True
+          Type:                  Ready
+        Deployment Refs:
+          demo-dc1-default-stargate-deployment
+        Progress:              Running
+        Ready Replicas:        1
+        Ready Replicas Ratio:  1/1
+        Replicas:              1
+        Service Ref:           demo-dc1-stargate-service
+        Updated Replicas:      1
+Events:                        <none>
+```
 
-To add the K8ssandra helm chart repos:
+### Extract credentials
 
-1. Install [Helm v3+](https://helm.sh/docs/intro/install/) if you haven't already.
+Use the following commands to extract the username and password:
 
-1. Add the main K8ssandra stable Helm chart repo:
+```bash
+CASS_USERNAME=$(kubectl get secret demo-superuser -n k8ssandra-operator -o=jsonpath='{.data.username}' | base64 --decode)
 
-    ```bash
-    helm repo add k8ssandra https://helm.k8ssandra.io/stable
-    ```
+echo $CASS_USERNAME
+```
 
-1. If you want to access K8ssandra services from outside of the Kubernetes cluster, also add the Traefik Ingress repo:
+**Output:**
+```bash
+demo-superuser
+```
 
-    ```bash
-    helm repo add traefik https://helm.traefik.io/traefik
-    ```
+Now obtain the password secret:
 
-1. Finally, update your helm repository listing:
+```bash
+CASS_PASSWORD=$(kubectl get secret demo-superuser -n k8ssandra-operator -o=jsonpath='{.data.password}' | base64 --decode)
 
-    ```bash
-    helm repo update
-    ```
+echo $CASS_PASSWORD
+```
 
+**Output example - your value will be different:**
+```bash
+ACK7dO9qpsghIme-wvfI
+```
 {{% alert title="Tip" color="success" %}}
-Alternatively, you can download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
+You'll use the extract credentials for subsequent authentication in deployed containers.
 {{% /alert %}}
 
-## Install K8ssandra
+### Verify cluster status
 
-The K8ssandra helm charts make installation a snap. You can override chart configurations during installation as necessary if you're an advanced user, or make changes after a default installation using `helm upgrade` at a later time.
+```bash
+kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
```

Comment:
```suggestion
kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- nodetool -u demo-superuser -pw 3CAvGWc4mRna8tODJgeN status
```

(I made this same comment below on another similar block)

This version of the command cleans up the output a bit and matches I think more what you want, it will directly run the status command and output it, without having to go into the shell interactively.

Reply:
Yes, but the -pw value should be `ACK7dO9qpsghIme-wvfI` from the result just above it.   So altogether, we have:


### Verify cluster status

```bash
kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- nodetool -u demo-superuser -pw ACK7dO9qpsghIme-wvfI status
```

**Output:**

```bash
Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
Datacenter: dc1
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens  Owns (effective)  Host ID                               Rack
UN  10.244.4.4  96.71 KiB  16      100.0%            bbdb6a9a-063b-4565-9704-f4caa6fd80f1  default
UN  10.244.2.5  96.72 KiB  16      100.0%            61dfa8cc-2a8b-4e8f-ae82-01c51833e0ba  default
UN  10.244.3.4  96.63 KiB  16      100.0%            369aa179-d96e-4f21-a893-f6e6dc84b396  default
```